src/SignalRServiceExtension/Config/SignalRConfigProvider.cs

```diff
@@ -61,12 +55,7 @@ public void Initialize(ExtensionConfigContext context)
                 logger.LogWarning($"Unsupported service transport type: {serviceTransportTypeStr}. Use default {options.AzureSignalRServiceTransportType} instead.");
             }
 
-            serviceManager = new ServiceManagerBuilder().WithOptions(o =>
-            {
-                o.ConnectionString = options.ConnectionString;
-                o.ServiceTransportType = options.AzureSignalRServiceTransportType;
-            }).Build();
-            StaticServiceHubContextStore.ServiceHubContextStore = new ServiceHubContextStore(serviceManager, loggerFactory);
+            StaticServiceHubContextStore.ServiceHubContextStore = new ServiceHubContextStore(null, loggerFactory);
```

Comment:
Is it possible that multiple Azure functions have different connection strings and sharing the same ServiceHubContextStore?

Reply:
Yes. It's better to make connectionString+hubName as key to ServiceHubContextStore?

Reply:
if there is such case, we need to make context store a dictionary of service managers.